### ISSUE TITLE
consider how to solve queue of thousands waiting PRs

### ISSUE THREAD

**Issue by matkoniecz** [freeCodeCamp/freeCodeCamp#34425]
Currently this project is a bit victim of its own success. 

It is obvious that it will not benefit strongly from more PRs as the bottleneck is clearly at processing them. Adding even more PRs to pile of over 8 thousand waiting has only marginal benefits.

It is sad for project that has "PRs welcome" in readme. With more than 8 thousands of open PRs waiting for handling (merging/rejecting) it is clear that this project would benefit from more maintainers that would process them.

I have nearly no knowledge about this project and I am unable to propose anyone specific. And anyway maybe the real problem is elsewhere. Maybe "more people allowed to merge PRs" overall will be harmful. But I want to point out obvious - there are hundreds (thousands?) of people waiting for their PR to be processed and it is a suboptimal situation.

EDIT: sorry for the initial pointlessly hostile form

EDIT2: for situation see https://github.com/freeCodeCamp/freeCodeCamp/pulse/monthly (currently it is at almost 2k merged PRs within last month and more than 6k PRs proposed)

EDIT3: I would be really happy to see this kind of problem in my project (seriously, even if that changes would be just fixing typos 2k of PRs in a month is amazing)

**Comment by mikeattara** [comment]
I really support this call.

**Comment by ezioda004** [comment]
Hi @matkoniecz,

Thanks for opening this issue.

You're right we have more than 8k+ PR open and that was due to Hacktoberfest influx. 
However, if you check the insights you'll see we've merged 2k PR's in the [past](https://github.com/freeCodeCamp/freeCodeCamp/pulse/monthly) month.

In the month of November, we have only received around 200 PR's which isnt a lot.

The PR cleanup will certainly take couple of months - but we are certain the new PR's that are being opened are meaningful.
